app/(home)/layout.js

```diff
@@ -0,0 +1,64 @@
+import "../globals.css";
+import { Inter } from "next/font/google";
+import Link from "next/link";
+import Image from "next/image";
+import { Button } from "@/components/ui/button";
+import { FaUserAlt } from "react-icons/fa";
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+};
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <body className={`${inter.className}`}>
+        <div className="flex justify-between flex-row items-center px-[15px] md:px-[30px] xl:px-[50px] 2xl:px-[100px] mx-auto py-[10px] z-50 w-full absolute top-0 left-0 right-0">
+          <div class="flex items-center">
```

Comment:
I think there is an error coming from these and some other divs where it says 'class' instead of 'className'